### PR TITLE
Share photo

### PR DESCRIPTION
**Pull request by johnmarksinclair**
## Purpose

TPY-22 Share photo to WhatsApp, Reddit, Instagram, etc.

## Proposed Change

Implement share functionality for final stylised image.

## Checklist

- [x] implemented share functionality
- [x] modified routing - split up take/ upload photo page and choose style page
- [x] various styling changes

### CODE REVIEW

File: screens/ChooseStyleScreen.tsx
Comment (DanielNugent) [inline screens/ChooseStyleScreen.tsx]: could add types for the function parameters so it knows what to expect

**Review by DanielNugent — DISMISSED** [review]

**Review by dabreadman — COMMENTED** [review]
I "worked" on it so it must be fine!

That being said there are a few things to add (as an issue):
- You can simply refuse to log in and still use that application as per usual
- Capture image buttons are off-screen in some Android devices (mine!)

Won't approve now since once you push the change requested by Daniel, you'd dismiss the approval
image

File: screens/ChooseStyleScreen.tsx
Comment (johnmarksinclair) [reply screens/ChooseStyleScreen.tsx]: done

File: screens/FinishedArtScreen.tsx
Comment (DanielNugent) [inline screens/FinishedArtScreen.tsx]: Could put in the actual types

```
import type { NativeStackScreenProps } from '@react-navigation/native-stack'

type RootStackParamList = {
  FinishedArtScreen: { image: string }
  ....
}

type Props = NativeStackScreenProps<RootStackParamList, 'FinishedArtScreen'>
```

https://reactnavigation.org/docs/typescript/

**Review by DanielNugent — APPROVED** [review]

**Pull request merged**
